source/includes/_10-one-time-sync-company-custom-check.html.md.erb

```diff
@@ -0,0 +1,547 @@
+# One-time Sync Company Custom 
+
+This section lists all endpoints that must be implemented for a one-time synchronous company custom check integration.
+
+One-time synchronous company custom checks are custom checks with the following behaviour:
+
+    * The check result will be returned directly upon request; the response will not be sent until the check is finished (synchronous)
+    
+    * The check operates at a point-in-time: there is no ongoing monitoring of the check result (one-time)
+
+
+## Get Check Configuration (One-time Sync Custom Company)
```

Comment:
Sentence case for subheadings. Eventually we'll move to sentence case for headings as well, but I think we should make that move all at once

"Get check configuration (One-time sync custom company)"